src/sdkRuntimeModels.ts

```diff
@@ -306,10 +306,11 @@ export interface SDKConfigApi {
 // FIXME: Resolve with User in @types/mparticle-web-sdk
 //        https://go.mparticle.com/work/SQDSDKS-5033
 export interface MParticleUser {
-    getMPID(): string;
-    getConsentState(): SDKConsentState;
-    getAllUserAttributes(): any; // FIXME;
-    getUserIdentities(): IdentityApiData; // FIXME: Is this correct?
+    getMPID?(): string;
+    getConsentState?(): SDKConsentState;
+    getAllUserAttributes?(): any; // FIXME;
+    getUserIdentities?(): IdentityApiData; // FIXME: Is this correct?
+    isLoggedIn?(): boolean;
```

Comment:
Why are these optional now?  

Also - can we turn the `FIXME`s into work items and link to them here?

Reply:
They're optional so that the mockBatchCreator in our tests doesn't complain. We already have a sustained eng task to refactor that but it's lower in urgency.

Reply:
They're optional so that the mockBatchCreator in our tests doesn't complain. We already have a sustained eng task to refactor that but it's lower in urgency.

Reply:
sgtm